webapp/channels/src/components/latex_block/latex_block.tsx

```diff
@@ -24,16 +26,15 @@ const LatexBlock = ({
         });
     }, []);
 
-    if (!enableLatex || katex === undefined) {
-        return (
-            <div
-                className='post-body--code tex'
-                data-testid='latex-disabled'
-            >
-                {content}
-            </div>
-        );
-    }
+    render(): React.ReactNode {
+        if (!this.props.enableLatex || this.state.katex === undefined) {
```

Comment:
@larkox is this a valid code in a functional component

Reply:
Oh! At least it is not "standard". Not sure how I missed it :sweat_smile: Thank you for pointing it out!

This should be changed to just return this instead of creating this render function

Reply:
Hmm, I am not able to understand the change. Could you be more specific? @larkox

Reply:
Never mind, got the issue. What happened is while I was doing my changes, at the same time the component got migrated from Class component to functional component. Hence, the code remained. Will push the fix in some time.